internal/config/funcs/doc.go

```diff
@@ -19,9 +19,9 @@ func Docs() map[string]string {
 		"dirname":         "return all except the last component of the string interpreted as a filesystem path",
 		"distinct":        "removes any duplicate elements from a list",
 		"element":         "return the value at given numeric index in the list",
-		"file":            "return the data in the file located at the given path",
-		"filebase64":      "return the data in the file located at the given path after base64 decoding it",
-		"fileexists":      "indicate if a file exists at the given path",
+		"file":            "return the data in the file located at the given path. This can be paired with the `${path.app}` variable to get a path relative to your app folder, i.e. `filebase64(\"${path.app}/file.txt\") string`\"",
+		"filebase64":      "return the data in the file located at the given path after base64 decoding it. This can be paired with the `${path.app}` variable to get a path relative to your app folder, i.e. `filebase64(\"${path.app}/file.txt\") string`",
+		"fileexists":      "indicate if a file exists at the given path. This can be paired with the `${path.app}` variable to get a path relative to your app folder, i.e. `filebase64(\"${path.app}/file.txt\") string`\"",
```

Comment:
I think there's a trailing `\"` that's not needed here...I think we can remove `string` too and move the backtick to the end of the function example?